pkg/transformer/utils.go

```diff
@@ -114,7 +116,14 @@ func ConfigAnnotations(service kobject.ServiceConfig) map[string]string {
 	for key, value := range service.Annotations {
 		annotations[key] = value
 	}
+	annotations["kompose.cmd"] = strings.Join(os.Args, " ")
+	version := exec.Command("kompose", "version")
```

Comment:
I don't like how we have an exec command here... but I don't think there's an alternative way to do this getting the actual Git hash...